katdal/test/test_chunkstore.py

```diff
@@ -18,10 +18,50 @@
 
 import numpy as np
 from numpy.testing import assert_array_equal
-from nose.tools import assert_raises
-
-from katdal.chunkstore import (ChunkStore, StoreUnavailable,
-                               ChunkNotFound, BadChunk)
+from nose.tools import assert_raises, assert_equal, assert_is_instance
+import dask.array as da
+
+from katdal.chunkstore import (ChunkStore, generate_chunks,
+                               StoreUnavailable, ChunkNotFound, BadChunk)
+
+
+def test_generate_chunks():
```

Comment:
This seems to be testing quite a lot of different things - maybe put it in a test class with multiple tests (at the very least, power-of-two could be a separate test).

Reply:
I would say it is testing one thing - `generate_chunks` - but I hear ya. I had proto-thoughts about splitting it up. My threshold is just higher :-)

Reply:
Done.